core/src/main/resources/hudson/model/Job/configure.jelly

```diff
@@ -26,50 +26,66 @@ THE SOFTWARE.
   Config page. derived class specific entries should go to configure-entries.jsp
 -->
 <?jelly escape-by-default='true'?>
-<j:jelly xmlns:j="jelly:core" xmlns:st="jelly:stapler" xmlns:l="/lib/layout" xmlns:f="/lib/form">
-
-  <l:layout type="one-column" permission="${it.EXTENDED_READ}" title="${%Config(it.displayName)}">
+<j:jelly xmlns:j="jelly:core" xmlns:st="jelly:stapler" xmlns:l="/lib/layout" xmlns:f="/lib/form" xmlns:p="/lib/hudson/project">
+  <l:layout permission="${it.EXTENDED_READ}" title="${%Config(it.displayName)}">
     <j:set var="readOnlyMode" value="${!it.hasPermission(it.CONFIGURE)}" />
 
-    <l:js src="jsbundles/config-scrollspy.js" />
-    <l:css src="jsbundles/config-scrollspy.css" />
+    <l:header>
+      <script src="${resURL}/jsbundles/section-to-sidebar-items.js" type="text/javascript" defer="true" />
+    </l:header>
```

Comment:
This introduces an inconsistency in the default installation with suggested plugins: the **Configure** project page uses a sidebar and `section-to-sidebar-items`, while the (suggested) [Folders](https://plugins.jenkins.io/cloudbees-folder/) and [Branch API](https://plugins.jenkins.io/branch-api/) plugins still use tabs and the (now deprecated) `config-scrollspy` in the **Configure** page for folders, multibranch pipelines, and organization folders. What is the plan for addressing this inconsistency?

Reply:
https://github.com/jenkinsci/cloudbees-folder-plugin/pull/237

Reply:
https://github.com/jenkinsci/workflow-job-plugin/pull/261

Reply:
Looking at the Branch API now...

Reply:
branch api just extends from the folder plugin as far as I can tell

Reply:
Perfect in that case! Just wrapping up the changes to the workflow job plugin..